### DISCUSSION TITLE
Make HTTPS the default protocol for links in the wysiwyg editor

### DISCUSSION THREAD

**Discussion by MiniDigger** [directus/directus#4288] [Feature Requests]
The insert link functionality in the wysiwyg editor has a neat likely function which warns you if you forget the protocol and asks you if you want to add it.

thats really cool, but prompted my users to add a bunch of http links to the website, which is really meh. 

I would propose to either make it insert https, or at least add two buttons for http and https.

(worth noting, I didnt had time to check if this is actually something directus controls or if that needs to be addressed elsewhere)

**Comment by rijkvanzanten** [comment]
This is default TinyMCE behavior. I think https://www.tiny.cloud/docs/plugins/opensource/link/#link_assume_external_targets controls this 🤔 

Mind opening a PR to set a good default for that option?

**Reply by MiniDigger** [reply]
it seems like directus uses the default value, false, which did promt me (cause I entered www) but didnt add https, but http.

the other options all assume all links are external, which is most likely not something we would want to force. what we would want would be a false option that does https. meh, ill bring it up with them

**Reply by MiniDigger** [reply]
actually, no https://www.tiny.cloud/docs/plugins/opensource/link/#link_default_protocol

"This option allows you to set a default protocol for links when inserting/editing a link via the link dialog. The protocol will apply to any links where the protocol has not been specified and the prefix prompt has been accepted."

so  link_default_protocol: 'https' is what I want. if this is something you want for directus itself, I can look at PRing that, I already use a custom wysiwyg interface with a bazillions of options and custom tinyemc plugins, so that would not help me.

**Reply by rijkvanzanten** [reply]
>  if this is something you want for directus itself, I can look at PRing that

Yeah that sounds like a the best default to me. Thanks!

**Reply by MiniDigger** [reply]
![image](https://user-images.githubusercontent.com/2185527/109187729-795cfd80-7792-11eb-83ce-5e623fb7d4cd.png)
 confirmed it

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
